design/hardware-classification-controller/support-for-new-parameters-hwcc.md

```diff
@@ -0,0 +1,243 @@
+# Add Support for extra hardware parameters in Disk, RAM, CPU and NIC
+ 
+## Table of Contents
+ 
+<!--ts-->
```

Comment:
We've mostly stopped using the table of contents in the other design documents. Did we fail to remove it from the template?

Reply:
Removed ##Table of Contents and added ##Status as per new template.
Thanks for correction.